openapi.yaml

```
@@ -1473,25 +1577,19 @@ components:
       title: wikiEntry
       type: object
       properties:
-        id:
-          type: string
         title:
           type: string
-        contents:
-          type: array
-          items:
-            type: object
-            properties:
-              type:
-                type: string
-                enum:
-                  - text
-                  - url
-              content:
-                type: string
+        content:
+          type: string
   headers: {}
   responses: {}
   parameters:
+    flow:
+      name: id
+      in: path
```

```suggestion
      in: query
```

Specify parameters to be query parameters, not path, so instead of /{id} it is actually ?id={id}. Same for other paths